File: libraries/hdm/functions/types/response/GetRPCMethodsResponseDTO.java
package hdm.functions.types.response;

import java.io.Serializable;

public class GetRPCMethodsResponseDTO
  implements ResponseDTO, Serializable
{
  private String[] methodList;

  public String[] getMethodList()
  {
    return this.methodList;
  }

  public void setMethodList(String[] methodList)
  {
    this.methodList = methodList;
  }
}

File: libraries/hdm/functions/types/response/GetRPCMethodsResponseDTO.js
/* @org.eclipse.vjet.dsf.resource.utils.CodeGen("VjoGenerator") */
vjo.ctype('hdm.functions.types.response.GetRPCMethodsResponseDTO') //< public
.satisfies('hdm.functions.types.response.ResponseDTO')
.satisfies('java.io.Serializable')
.protos({
    methodList:null, //< private String[]
    //> public String[] getMethodList()
    getMethodList:function(){
        return this.methodList;
    },
    //> public void setMethodList(String[] methodList)
    setMethodList:function(methodList){
        this.methodList=methodList;
    }
})
.endType();